java/pom.xml

```diff
@@ -4,6 +4,11 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <properties>
+        <maven.compiler.target>9</maven.compiler.target>
```

Comment:
I suspect a large number of corporate users are still targeting java 8, and haven't yet made the jump to Java 9. I'd love to be wrong.